### DISCUSSION TITLE
Пешеходный квест в Валенсии

### DISCUSSION THREAD

**Discussion by vitalynec** [tech-connect-meetups/tech_vlc#7] [Ideas]
Несколько раз слышал, что мы хотим познакомить наше коммьюнити с городом. В связи с чем появилась идея в кооперации с организаторами пешеходных квестов в Валенсии провести закрытый квест для участников коммьюнити. Можно использовать готовую программу или подумать на счет кастом-точек, специфичных для нашей аудитории.

**Comment by hola-asya** [comment]
Конвертировала [в волонтерскую задачку](https://github.com/tech-connect-meetups/tech_vlc/issues/8), классная идея!
